dashboard/app/models/script.rb

```diff
@@ -661,6 +661,39 @@ def link
     Rails.application.routes.url_helpers.script_path(self)
   end
 
+  def can_be_instructor?(user)
+    # If unit is in a unit group then decide based on unit group audience
+    return unit_group.can_be_instructor?(user) if unit_group
+
+    return false if user.student?
+    return true if user.permission?(UserPermission::CODE_INSTRUCTOR) || user.permission?(UserPermission::LEVELBUILDER)
+
+    if instructor_audience == 'plc_reviewer'
+      return user.permission?(UserPermission::PLC_REVIEWER)
+    elsif instructor_audience == 'facilitator'
+      return user.permission?(UserPermission::FACILITATOR)
+    elsif instructor_audience == 'teacher'
```

Comment:
Could this line just be `else` without specifying audience and then you could remove `return false if user.student?` and the final `false` return at the end of the method?  Only teachers can be instructors if the audience is teacher or student, right?